issue_metrics.py

```diff
@@ -193,14 +193,12 @@ def get_per_issue_metrics(
                 )
             if env_vars.hide_time_to_answer is False:
                 issue_with_metrics.time_to_answer = measure_time_to_answer(issue)
-            if env_vars.hide_time_to_close is False:
-                if issue["closedAt"]:
-                    issue_with_metrics.time_to_close = measure_time_to_close(
-                        None, issue
-                    )
-                    num_issues_closed += 1
-                else:
-                    num_issues_open += 1
+            if env_vars.hide_time_to_close is False and issue["closedAt"]:
+                issue_with_metrics.time_to_close = measure_time_to_close(None, issue)
+            if issue["closedAt"]:
+                num_issues_closed += 1
+            else:
+                num_issues_open += 1
```

Comment:
Logically equivalent but is this more readable/performant?
```suggestion
            if issue["closedAt"]:
                num_issues_closed += 1
                if env_vars.hide_time_to_close:
                    issue_with_metrics.time_to_close = measure_time_to_close(None, issue)
            else:
                num_issues_open += 1
```

Reply:
`if env_vars.hide_time_to_close is False` should be `if not env_vars.hide_time_to_close` IMO.  wdyt?

Reply:
I'm good with your code suggestion change, so we can track the `num_issues_closed` now but suggest the code edit above.